app/mailers/api_entreprise/user_mailer.rb

```diff
@@ -13,7 +13,7 @@ def transfer_ownership(old_owner, new_owner)
 
   def notify_datapass_for_data_reconciliation(user)
     @user = user
-    @authorization_requests_ids = user.tokens.pluck(:authorization_request_id).map(&:to_i)
+    @datapass_ids = user.authorization_requests.map(&:external_id).map(&:to_i)
```

Comment:
```suggestion
    @datapass_ids = user.authorization_requests.pluck(:external_id).compact
```

Reply:
ActiveRecord::StatementInvalid:
       PG::InvalidColumnReference: ERROR:  for SELECT DISTINCT, ORDER BY expressions must appear in select list
       LINE 1: ...horization_request_roles"."user_id" = $1 ORDER BY "authoriza...

Reply:
alkdhwqaldhasdlh ce foutu `default_scope`

Reply:
On a un default scope? `ag default_scope` me retourne rien

Reply:
Laisse tomber on va harden l'intégrité